### DISCUSSION TITLE
How do I reinstall Stellarium and open the locked features?

### DISCUSSION THREAD

**Discussion by mrlocker48** [community/community#84952] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

 I paid $20 about 2 years ago.  I've reinstalled the program with no opportunity to open the locked features with my paid entry.

**Comment by vivekvardhan2810** [comment]
If you have a legitimate license for Stellarium, and you're having trouble accessing the features you paid for after reinstalling the program, here are some steps you can take:

1.  Check Your License Information:
   - Locate the email or documentation that contains your license key or any information related to your purchase.
   - Make sure you are using the correct license key or activation method.

2. Contact Stellarium Support:
   - Visit the official Stellarium website and look for a support or contact page.
   - Reach out to their support team, explaining the issue and providing them with your purchase details.
   - They should be able to assist you in resolving the problem or providing a new license key if necessary.

3.  Reinstall with the Correct Version:
   - Ensure that you are installing the correct version of Stellarium that corresponds to your purchased license. Sometimes, different versions may have different licensing mechanisms.

4. License Activation:
   - Check if there is a specific activation process for your license. Some software requires online activation or manual input of the license key.

5. Check for Updates:
   - After reinstalling, make sure you have the latest version of Stellarium. Updates may fix bugs or address licensing issues.

Remember to approach this situation responsibly and avoid using any unauthorized methods to unlock features. If you've legitimately purchased a license and are experiencing difficulties, the software provider's support team should be able to assist you in resolving the issue.
